docker/cmsmon-rucio-ds-mongo/Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM mongo as builder
+FROM registry.cern.ch/cmsmonitoring/cmsmon-hadoop-base:spark3-20220606
+MAINTAINER Ceyhun Uzunoglu ceyhunuzngl@gmail.com
+
+ENV WDIR=/data
+ENV HADOOP_CONF_DIR=/etc/hadoop/conf
+# mongoimport is in WDIR
+ENV PATH="${PATH}:/usr/hdp/hadoop/bin/hadoop:/usr/hdp/spark3/bin:/usr/hdp/sqoop/bin:${WDIR}/CMSSpark/bin:${WDIR}"
+# CMSMonitoring folder is in WDIR
+ENV PYTHONPATH "${PYTHONPATH}:${WDIR}:${WDIR}/CMSSpark/src/python"
+# LCG101 uses this cvmfs python in Spark worker nodes
+ENV PYSPARK_PYTHON=/cvmfs/sft.cern.ch/lcg/releases/Python/3.9.6-b0f98/x86_64-centos7-gcc8-opt/bin/python3
+# Local
+ENV PYSPARK_DRIVER_PYTHON=/usr/bin/python
+
+COPY --from=builder /usr/bin/mongoimport $WDIR
+
+RUN git clone https://github.com/dmwm/CMSSpark.git && \
```

Comment:
I suggest to use specific tag of the code since master branch can be unstable.

Reply:
Done